test/parallel/test-regress-GH-2245.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable strict */
```

Comment:
Sorry, I could have been clearer above, the issue here is that this code is not valid in strict mode, since `eval` in strict mode does not introduce new variables into the surrounding scope. Therefore this file cannot have strict mode enabled, and hence this linter rule.